examples/integration_test.py

```diff
@@ -1217,6 +1327,10 @@ def print_usage(exitcode, reason=None):
         print('=' * 30, 'Verifying Admin API', '=' * 30)
         verify_admin()
 
+    if 'avro-HTTPS' in modes:
+        print('=' * 30, 'Verifying AVRO with https', '=' * 30)
```

Comment:
So.. I wanted https to be HTTPS in the print, since it is an acronym, but https to remain in lower-case for the test mode.